excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -512,7 +512,7 @@ public void caseFieldEntityInvalidUserRole_createErrorMessageCalled_customMessag
         final CaseTypeEntity caseTypeEntity = caseTypeEntity("case type");
         final CaseFieldEntity caseFieldEntity = caseFieldEntity(
             "case field", SecurityClassification.RESTRICTED);
-        final CaseFieldACLEntity entity = caseFieldUserRoleEntity("crud");
+        final CaseFieldACLEntity entity = caseFieldACLEntity("crud");
 
         final CaseFieldEntityInvalidUserRoleValidationError error = new CaseFieldEntityInvalidUserRoleValidationError(
```

Comment:
Rename test when validation error `CaseFieldEntityInvalidUserRoleValidationError` is renamed